pkg/component/application/slack/v0/.compogen/extra-setup.mdx

```diff
@@ -0,0 +1,13 @@
+### Connecting through a Slack App
+
+The Slack component connects with your workspace via a [Slack
+App](https://api.slack.com/docs/apps). In order to read / write messages from / to Slack, you'll need to:
+
+- [Add the
+  app](https://slack.com/intl/en-gb/help/articles/202035138-Add-apps-to-your-Slack-workspace)
+  to your workspace.
+- Add the app to the channels you want to interact with.
+
+On **Instill Cloud**, the `instill-ai` app will be used fo connect with Slack.
```

Comment:
@EiffelFly when the app is published it'd be good to add a reference to the Slack App marketplace

Reply:
yes indeed, I will add the ticket